mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/MavenPropertyResolver.java

```diff
@@ -18,4 +23,13 @@ static private boolean isSkip() {
         return Boolean.valueOf(System.getProperty("maven.test.skip", "false"));
     }
 
+    static boolean isSpecificTestClassSet() {
+        String specificTestClasses = System.getProperty("test");
+        return specificTestClasses != null && !containsPattern(specificTestClasses);
+    }
+
+    private static boolean containsPattern(String specificTestClasses) {
+        Matcher matcher = TEST_CLASS_PATTERN.matcher(specificTestClasses);
```

Comment:
I thought we disable the extension if only a single test needs to be executed. 
When "two tests" given we could still have smart-testing enabled. 

Example: when I have smart testing enabled with suppose `selecting` mode and I specify two tests. If only one of those fulfills the criteria, only one be executed.  

What do you suggest ? (I'll fix it if you suggest otherwise.)